README.md

```diff
@@ -189,34 +192,85 @@ If you don't need a controller to handle a static component, you can route direc
 inertia 'about' => 'AboutComponent'
 ```
 
-### SSR
+### SSR _(experimental)_
+
+Enable SSR via the configuration options for [`ssr_enabled`](#ssr_enabled-experimental) and [`ssr_url`](#ssr_url-experimental).
+
+When using SSR, don't forget to add `<%= inertia_ssr_head %>` to the `<head>` of your layout (i.e. `application.html.erb`).
 
-Enable SSR via the config settings for `ssr_enabled` and `ssr_url`.
+## Configuration ⚙️
 
-When using SSR, don't forget to add `<%= inertia_headers %>` to the `<head>` of your `application.html.erb`.
+Inertia Rails can be configured globally or in a specific controller (and subclasses).
 
-## Configuration
+### Global Configuration
 
-Inertia Rails has a few different configuration options that can be set anywhere, but the most common location is from within an initializer.
+If using global configuration, we recommend you place the code inside an initializer:
 
-The default config is shown below
 ```ruby
+# config/initializers/inertia.rb
+
 InertiaRails.configure do |config|
-  
-  # set the current version for automatic asset refreshing. A string value should be used if any.
-  config.version = nil
-  # enable default inertia rendering (warning! this will override rails default rendering behavior)
-  config.default_render = true
-  
-  # ssr specific options
-  config.ssr_enabled = false
-  config.ssr_url = 'http://localhost:13714'
+  # Example: force a full-reload if the deployed assets change.
+  config.version = ViteRuby.digest
+end
+```
 
-  config.deep_merge_shared_data = false
-  
+The default configuration can be found [here](https://github.com/inertiajs/inertia-rails/blob/master/lib/inertia_rails/configuration.rb#L5-L22).
+
+### Local Configuration
+
+Use `inertia_config` in your controllers to override global settings:
+
+```ruby
+class EventsController < ApplicationController
+  inertia_config(
+    version: "events-#{InertiaRails.configuration.version}",
```

Comment:
Local version configuration seems like a bit of a footgun. In this example, won't you trigger a hard refresh every time you visit this controller from a different controller since those versions will always be mismatched?

All the other configuration options make perfect sense to optionally configure per-controller. Maybe we should exclude `version` (and only `version`) from being locally configured?

Reply:
> won't you trigger a hard refresh every time you visit this controller from a different controller since those versions will always be mismatched?

You could have different apps, with entirely different Vite bundles, so that would actually make sense and the hard refresh would "make it work". Linking across apps using Inertia navigations isn't "recommended", but it's nice that it would work out of the box.

Having an exception for `version` seems unnecessary, it's completely optional to use it, and in some cases it's actually useful.